### PR TITLE
refactor(api-calls): Pull static methods out of Wrapper class

### PR DESCRIPTION
**Pull request by JonathanXITASO**
# Description

Just a small change to get rid of a Util class and export the static functions directly

Fixes # (MNES-1223)

## Type of change

-   [x] Minor change (non-breaking change, e.g. documentation adaption)

# Checklist:

-   [x] I have performed a self-review of my code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation
-   [x] My changes generate no new warnings
-   [ ] I have added tests that prove my fix or my feature works
-   [x] New and existing tests pass locally with my changes
-   [x] My changes contain no console logs

### CODE REVIEW

**Review by GailMelanie — APPROVED** [review]

**Comment by fleanegan** [comment]
Nice work. I really like the names and only have one nitpicky remark: Normally we write function names in lower case

File: src/lib/services/apiResponseWrapper.ts [hunk 1]
```diff
@@ -33,37 +33,35 @@ export type ApiResponseWrapperError<T> = {
...
-        } else {
-            return {
-                isSuccess: false,
-                result: await response.json(),
-                errorCode: status,
-                message: response.statusText,
-            };
-        }
-    }
+export function WrapErrorCode<T>(error: ApiResultStatus, message: string): ApiResponseWrapperError<T> {
```
Comment (FranzXitaso) [inline src/lib/services/apiResponseWrapper.ts:36]: As already mentioned by Frederik, I'd write the function names lowercase. Apart from that, I like the function names a lot.

**Review by FranzXitaso — COMMENTED** [review]
Very nice! Apart from writing function names in lowercase, I like this a lot!

**Review by FranzXitaso — APPROVED** [review]
nice work!

**Pull request merged**